ci-operator/step-registry/openshift/microshift/infra/sos-aws/openshift-microshift-infra-sos-aws-commands.sh

```diff
@@ -6,6 +6,142 @@ source "${SHARED_DIR}/ci-functions.sh"
+        cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+<testsuite name="cluster install" tests="5" failures="2">
+    <testcase name="install should succeed: cluster bootstrap"/>
+    <testcase name="install should succeed: infrastructure"/>
+    <testcase name="install should succeed: other"/>
+    <testcase name="install should succeed: configuration">
+        <failure message="">Failed to configure host for MicroShift conformance</failure>
+    </testcase>
+    <testcase name="install should succeed: overall">
+        <failure message="">MicroShift install failed overall</failure>
+    </testcase>
+EOF
+    elif [ "$EXIT_CODE" == "$EXIT_CODE_PCP_FAILURE" ]; then
+        cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+<testsuite name="cluster install" tests="5" failures="2">
+    <testcase name="install should succeed: other"/>
+    <testcase name="install should succeed: infrastructure"/>
+    <testcase name="install should succeed: cluster bootstrap"/>
+    <testcase name="install should succeed: configuration">
+        <failure message="">Failed to install pcp in MicroShift VM</failure>
+    </testcase>
+    <testcase name="install should succeed: overall">
+        <failure message="">MicroShift install failed overall</failure>
+    </testcase>
+EOF
+    elif [ "$EXIT_CODE" == "$EXIT_CODE_WAIT_CLUSTER_FAILURE" ]; then
+        cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+<testsuite name="cluster install" tests="5" failures="2">
+    <testcase name="install should succeed: other"/>
+    <testcase name="install should succeed: infrastructure"/>
+    <testcase name="install should succeed: cluster bootstrap"/>
+    <testcase name="install should succeed: configuration">
+        <failure message="">Failed to install pcp in MicroShift VM</failure>
+    </testcase>
+    <testcase name="install should succeed: overall">
+        <failure message="">MicroShift install failed overall</failure>
+    </testcase>
+EOF
+    else
+      cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+      <testsuite name="cluster install" tests="2" failures="2">
+        <testcase name="install should succeed: other">
+          <failure message="">MicroShift cluster install failed with other errors</failure>
+        </testcase>
+        <testcase name="install should succeed: overall">
+          <failure message="">MicroShift cluster install failed overall</failure>
+        </testcase>
+EOF
+    fi
+    cat >>"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+</testsuite>
+EOF
+}
+
+IP_ADDRESS="$(cat "${SHARED_DIR}/public_address")"
+HOST_USER="$(cat "${SHARED_DIR}/ssh_user")"
+INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
+
+echo "Using Host $IP_ADDRESS"
+
+mkdir -p "${HOME}/.ssh"
+cat <<EOF >"${HOME}/.ssh/config"
+Host ${IP_ADDRESS}
+  User ${HOST_USER}
+  IdentityFile ${CLUSTER_PROFILE_DIR}/ssh-privatekey
+  StrictHostKeyChecking accept-new
+  ServerAliveInterval 30
+  ServerAliveCountMax 1200
+EOF
+chmod 0600 "${HOME}/.ssh/config"
+
```

Comment:
This should not be necessary because we have the `ci_script_prologue` call above

Reply:
You are right, must have missed it. Done.